doc/README.md

```diff
@@ -198,14 +198,14 @@ paths with the highest cum weight.
 
 *  **_cum_ value**, the value of a location plus all of its descendants, is indicated by the nodes' color.
 
-    *   Redder nodes have larger total values.
-    *   Greyer nodes have smaller total values.
+    *   Redder nodes have larger _cum_ values.
```

Comment:
I think for numbers it's more common to use "greater" than "larger".

@nolanmar511, what do you think?

Reply:
Done. I think so too.